src/openms/include/OpenMS/ANALYSIS/MAPMATCHING/QTClusterFinder.h

```diff
@@ -142,6 +142,9 @@ namespace OpenMS
     /// Set of features already used
     std::unordered_set<const OpenMS::GridFeature*> already_used_;
 
+    ///
+    std::map<double, double> bin_tolerances_;
```

Comment:
is this mapping with double dangerous?

Reply:
Since I only use lower_bound and don't really care in which of the corner case bins it falls, I don't think so.

Reply:
ok if it can't happen that it falls into the one after last bin it should be fine